src/tests/unittests/validation/FramebufferValidationTests.cpp

```diff
@@ -77,6 +77,13 @@ TEST_F(FramebufferValidationTest, BasicWithEmptyAttachment) {
         .SetRenderPass(renderpass)
         .SetDimensions(100, 100)
         .GetResult();
+
+    // set clear color value for attachment 0
+    framebuffer.AttachmentSetClearColor(0, 0.0f, 0.0f, 0.0f, 0.0f);
```

Comment:
This test shouldn't be valid anymore and we should force framebuffers to have all attachment defined. Also we should test setting clear colors in a different test.